common4j/src/main/com/microsoft/identity/common/java/cache/SharedPreferencesAccountCredentialCache.java

```diff
@@ -96,7 +96,6 @@ public SharedPreferencesAccountCredentialCache(
         super(sharedPreferencesFileManager);
         Logger.verbose(TAG, "Init: " + TAG);
         mCacheValueDelegate = accountCacheValueDelegate;
-        new Thread(this::removeSha1ApplicationIdentifierAccessTokensIfNeeded).start();
```

Comment:
Is this part of reverting token removal logic?

Reply:
yes, that thread ran the logic to remove the token if necessary.